### PR TITLE
Redirect http to https

### PR DESCRIPTION
**Pull request by m-wynn**
This will make things super secure.

### CODE REVIEW

File: roles/jekyll/files/jekyll.conf [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <VirtualHost lug.utdallas.edu:80>
 	ServerName lug.utdallas.edu
-	DocumentRoot /var/www/home/
+            Redirect permanent / https://lug.utdallas.edu
```
Comment (the-maldridge) [inline roles/jekyll/files/jekyll.conf]: Please fix this indentation, this is not in line with the style of the rest of the repo.

**Comment by m-wynn** [comment]
I have fixed that.

**Comment by the-maldridge** [comment]
No, you haven't

**Comment by the-maldridge** [comment]
Your change is being applied, should be live in a few minutes.

**Pull request merged**